File: 0709-to-lower-case/0709-to-lower-case.cpp
class Solution {
public:
    string toLowerCase(string s) {
        int n = s.length();
        for (int i = 0; i < n; i++) {
            s[i] = tolower(s[i]);
        }
        return s;
    }
};


// class Solution {
// public:
//     string toLowerCase(string s) {
//         transform(s.begin(), s.end(), s.begin(), ::tolower);
//         return s;
//     }
// };

File: LeetCode/0709-to-lower-case/0709-to-lower-case.java
class Solution {
    public String toLowerCase(String s) {
        StringBuilder str = new StringBuilder();
        for (char c : s.toCharArray()) {
            str.append(Character.toLowerCase(c));
        }
        return str.toString();
    }
}


// class Solution {
//     public String toLowerCase(String s) {
//         return s.toLowerCase();
//     }
// }